api.md

```diff
@@ -234,6 +236,33 @@ Response
 }
 ```
 
+<a name=device-authorized-post></div>
+
+#### `POST /api/v2/device/:deviceID/authorized` - authorize a device
+
+Marks a device as authorized, for Tailnets where device authorization is required.
+
+##### Parameters
+
+###### POST Body
+`authorized` - whether the device is authorized; only `true` is currently supported
+```
+{
+  "authorized": true
+}
+```
+
+##### Example
+
+```
+curl 'https://api.tailscale.com/api/v2/device/11055/authorized' \
+-u "tskey-yourapikey123:" \
+--data-binary '{"authorized": true}'
+```
+
+The response is 2xx on success. The body is currently an empty JSON
```

Comment:
This last part is a bit weird but I'm not sure what's normal here. Return a 204 (no content) status code? Return the Device object itself after mutation? Return just the authorization status? But that's always true on success.

For now it's just an empty JSON object.